src/main/java/org/apache/cassandra/sidecar/CassandraSidecarDaemon.java

```diff
@@ -38,19 +38,22 @@
     private static final Logger logger = LoggerFactory.getLogger(CassandraSidecarDaemon.class);
     private final HttpServer server;
     private final Configuration config;
+    private final CDCReaderService cdcReaderService;
 
     @Inject
-    public CassandraSidecarDaemon(HttpServer server, Configuration config)
+    public CassandraSidecarDaemon(HttpServer server, Configuration config, CDCReaderService cdcReaderService)
     {
         this.server = server;
         this.config = config;
+        this.cdcReaderService = cdcReaderService;
     }
 
     public void start()
     {
         banner(System.out);
         validate();
         logger.info("Starting Cassandra Sidecar on {}:{}", config.getHost(), config.getPort());
+        cdcReaderService.start();
```

Comment:
and I also think we can add a common method where all other service can be added inner the method . 
such as startInitService() , for me ,I think cdcReaderService is a sidecar init start service and we can use a flag to 
show the service's start or not .